parsec-3.0/htu.txt

```diff
@@ -0,0 +1,20 @@
+To uncompress a workload, the command is :
+
+- FOR PARSEC/SPLASH2 BENCHMARKS : 
```

Comment:
Because as benchmarks from splash2 and splash2x have same names and are in different folders, I wanted to be possible to specify which input the user want to uncompress, the one from splash2 or the one from splash2x